src/app/home/cardsForMetaCampaigns.ts

```diff
@@ -93,7 +93,7 @@ export const cardsForMetaCampaigns = function (
         headerText: 'Applications for Christmas Challenge now open!',
         backgroundImageUrl: new URL('/assets/images/card-background-cc-lights.jpg', donateUriPrefix),
         iconColor: 'brand-1',
-        bodyText: 'Deadline is 7 July 2023',
+        bodyText: 'Deadline is 19th July 2023',
```

Comment:
```suggestion
        bodyText: 'Deadline is 19 July 2023',
```

@dorota-joanna I suggest we leave out the suffix for consistency with the card next to this.